mongodbatlas_test.go

```diff
@@ -263,7 +263,7 @@ func testCredsExists(projectID, publicKey, privateKey, username string) (err err
 		return
 	}
 
-	_, _, err = client.DatabaseUsers.Get(context.Background(), projectID, username)
+	_, _, err = client.DatabaseUsers.Get(context.Background(), "admin", projectID, username)
```

Comment:
We could get the `database name` as an argument too, not really a fan or hard coding in but in the other hand, we could have a lot of arguments for this function too.